Makefile

```diff
@@ -9,6 +10,11 @@ lint:
 		cargo clippy --all --all-targets --all-features --no-deps -- --deny warnings
 		cargo fmt --all -- --check
 
+lint-optional:
```

Comment:
I would add a comment here explaining this lint - why these lint levels are left out of the main lint config